src/public/modules/forms/base/components/field-attachment.client.component.js

```diff
@@ -22,6 +22,8 @@ angular.module('forms').component('attachmentFieldComponent', {
 function attachmentFieldComponentController($timeout) {
   const vm = this
 
+  vm.MB = MB
```

Comment:
why do we need this?

Reply:
some calculations done in the html counterpart

```
ngf-max-size="{{ vm.field.attachmentSize * vm.MB }}"
```

We can theoretically also precalculate this in the JS file but that will probably require watching etc

Reply:
ah I see it now, thanks!